src/controllers/controller.radar.js

```diff
@@ -50,6 +51,7 @@ module.exports = function(Chart) {
 				// Model
 				_model: {
 					// Appearance
+					spanGaps: dataset.spanGaps ? dataset.spanGaps : me.chart.options.spanGaps,
```

Comment:
`dataset.spanGaps !== undefined ? dataset.spanGaps : me.chart.options.spanGaps`, else it will ignore `dataset.spanGaps: false`